support-workers/src/main/scala/com/gu/emailservices/SubscriptionEmailFieldHelpers.scala

```diff
@@ -30,14 +30,13 @@ object SubscriptionEmailFieldHelpers {
 
   def describe(
     paymentSchedule: PaymentSchedule,
-    billingPeriod: BillingPeriod,
-    currency: Currency,
+    product: ProductType,
     promotion: Option[Promotion],
-    isGift: Boolean = false
+    fixedTerm: Boolean = false
```

Comment:
this should be `fixedTerm`, because it determines the wording of the message saying how much you'll pay and when.